src/google/cloud/ndb/model.py

```diff
@@ -4082,33 +4082,95 @@ def non_transactional(*args, **kwargs):
     raise NotImplementedError
 
 
-def get_multi_async(*args, **kwargs):
-    raise NotImplementedError
+def get_multi_async(keys, **options):
+    """Fetches a sequence of keys.
 
+    Args:
+        keys (Sequence[:class:`~google.cloud.ndb.key.Key`]): A sequence of keys.
+        **options (Dict[str, Any]): The options for the request. For example,
+            ``{"read_consistency": EVENTUAL}``.
+    Returns:
+        List[:class:`~google.cloud.ndb.tasklets.Future`]: List of futures.
+    """
+    return [key.get_async(**options) for key in keys]
```

Comment:
Would it make sense to use a MultiFuture here?

Reply:
Never mind this and similar comments. I see the NDB API does return multiple futures, which is not what I would have done, but is what the caller is expecting.